build_resolvers/pubspec.yaml

```diff
@@ -20,19 +20,3 @@ dependencies:
 
 dev_dependencies:
   test: ^1.0.0
-  build_test: ^1.0.0
```

Comment:
[nit] Deleting non-overrides should be mentioned in the commit message.

Reply:
will update when I merge